donut/modules/feedback/routes.py

```diff
@@ -15,7 +15,7 @@
 
 @blueprint.route('/feedback/<group>')
 def feedback(group):
-    if group not in Groups:
+    if group not in Groups or not auth_utils.is_caltech_user():
         return flask.abort(404)
```

Comment:
This should show a "you need to login" message, not a 404 error. See how the directory search pages do it: https://github.com/ASCIT/donut/blob/3181076b26c91db50177924794b1309af0572f04/donut/modules/directory_search/routes.py#L15-L16